packages/app-degree-pages/src/components/DetailPage/components/WhyChooseAsu/index.js

```diff
@@ -16,7 +18,7 @@ const WhyChooseAsu = ({ sectionIntroText, cards, defaultCards }) => {
   return (
     <section className="container" data-testid="why-choose-asu">
       <h2>Why choose ASU</h2>
-      <p>{sectionIntroText}</p>
+      <p dangerouslySetInnerHTML={sanitizeDangerousMarkup(sectionIntroText)} />
```

Comment:
@maycomayco Should this be inside a `<div>` tag instead of a `<p>` tag since the markup lis likely to include p tags already?

Reply:
fixed